### PR TITLE
CTOR-601 Refresh applications-docker-restapi

### PR DESCRIPTION
**Pull request by lucie-tirand**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 1]
```diff
@@ -69,72 +101,343 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+dnf install centreon-pack-applications-docker-restapi
+```
 
-1. Install the plugin on every poller expected to monitor Docker resources:
+</TabItem>
+<TabItem value="Debian 11 & 12"" label="Debian 11 & 12"">
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-docker-restapi.md:130]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: pp/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 2]
```diff
@@ -69,72 +101,343 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-pack-applications-docker-restapi
+dnf install centreon-plugin-Applications-Docker-Restapi
 ```
 
-3. Install the "Docker-Restapi" Centreon Monitoring Connector from the **Configuration > Monitoring Connector Manager** page
+</TabItem>
+<TabItem value="Debian 11 & 12"" label="Debian 11 & 12"">
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-docker-restapi.md:177]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 3]
```diff
@@ -67,72 +102,343 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
---hostname='192.168.0.50' \
---port='2375'   \
---warning-node-status='' \
---critical-node-status='%{status} !~ /ready/ || %{manager_status} !~ /reachable|-/' \
---verbose
-
-OK: Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0 | 'containers_running'=7;;;0; 'containers_stopped'=2;;;0; 'containers_paused'=0;;;0;
-Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0
+/usr/lib/centreon/plugins/centreon_docker_restapi.pl \
+	--plugin=cloud::docker::restapi::plugin \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md:267]: is the example ok?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 4]
```diff
@@ -67,72 +102,343 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
+	--warning-traffic-out='' \
+	--critical-traffic-out='' \
+	--verbose
 ```
 
-La commande ci-dessus requête un noeud Docker (```--mode=node-status```) sur le port 2375 (```--port='2375'```) et fournit son état ('ready' ou pas) ainsi que l'état de ses containers en état "running","stopped" et "paused".
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All containers are ok All container traffics are ok | '*containers*#container.cpu.utilization.percentage'=%;;;0;100'*containers*#container.memory.usage.bytes'=B;;;;'*containers*#container.disk.io.read.usage.iops'=iops;;;0;'*containers*#container.disk.io.write.usage.iops'=iops;;;0;'*containers_traffic*#container.traffic.in.bitspersecond'=b/s;;;0;'*containers_traffic*#container.traffic.out.bitspersecond'=b/s;;;0;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md:294]: and the output?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md (hunk 3 above)
Comment (lucie-dubrunfaut) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md:267]: Let's put the old one (I can't test this locally)

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 5]
```diff
@@ -67,72 +102,329 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
---plugin=cloud::docker::restapi::plugin \
---mode=node-status \
---hostname='192.168.0.50' \
---port='2375'   \
---warning-node-status='' \
---critical-node-status='%{status} !~ /ready/ || %{manager_status} !~ /reachable|-/' \
---verbose
+/usr/lib/centreon/plugins/centreon_docker_restapi.pl \
+	--plugin=cloud::docker::restapi::plugin \
+	--mode=mode=node-status \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md:268]:
```suggestion
	--mode=node-status \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md [hunk 6]
```diff
@@ -67,72 +102,329 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
...
+	--warning-node-status='' \
+	--critical-node-status='%{status} !~ /ready/ || %{manager_status} !~ /reachable|-/' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
 
+```bash
 OK: Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0 | 'containers_running'=7;;;0; 'containers_stopped'=2;;;0; 'containers_paused'=0;;;0;
 Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md:280]:
```suggestion
OK: Node '10.0.0.1' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0 | 'containers_running'=7;;;0; 'containers_stopped'=2;;;0; 'containers_paused'=0;;;0;
Node '10.0.0.1' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0
```

**Comment by lucie-dubrunfaut** [comment]
[https://centreon.atlassian.net/browse/CTOR-878](CTOR-878)

Bonjour, sur cette page vous écrivez : "Modify the line starting with ExecStart as shown below:
ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock -H=tcp://0.0.0.0:2375"
Or cette configuration induit un GROS problème de sécurité. Pour citer cette page : https://docs.docker.com/reference/cli/dockerd/ "You could set it to 0.0.0.0:2375 or a specific host IP to give access to everybody, but that isn't recommended because someone could gain root access to the host where the daemon is running."
Il faudrait à minima alerter l'utilisateur sur l'utilisation de cette configuration et recommander de se baser sur la documentation de Docker.

**Review by cg-tw — APPROVED** [review]

**Pull request merged**